suite-common/wallet-core/src/accounts/constants.ts

```diff
@@ -0,0 +1 @@
+export const modulePrefix = '@suite-common/wallet-core';
```

Comment:
this is not module prefix anymore, please name it just `const actionPrefix = '@common/wallet-core/accounts'`